gateway-ha/src/main/java/io/trino/gateway/ha/resource/GatewayWebAppResource.java

```diff
@@ -160,7 +163,7 @@ public Response getDistribution(QueryDistributionRequest query)
         Map<String, List<DistributionResponse.LineChart>> lineChartMap = lineChart.stream().collect(Collectors.groupingBy(DistributionResponse.LineChart::getName));
         List<DistributionResponse.DistributionChart> distributionChart = lineChartMap.values().stream().map(d -> {
             DistributionResponse.DistributionChart dc = new DistributionResponse.DistributionChart();
-            DistributionResponse.LineChart lc = d.get(0);
+            DistributionResponse.LineChart lc = d.getFirst();
```

Comment:
Please separate a commit. This change is unrelated to "Fixed bug in UI where non indicative banner appears in routing rules page for external service".